.env

```diff
@@ -86,4 +86,6 @@ _APP_GRAPHQL_MAX_COMPLEXITY=250
 _APP_GRAPHQL_MAX_DEPTH=3
 DOCKERHUB_PULL_USERNAME=
 DOCKERHUB_PULL_PASSWORD=
-DOCKERHUB_PULL_EMAIL=
\ No newline at end of file
+DOCKERHUB_PULL_EMAIL=
+_APP_GITHUB_PRIVATE_KEY=
```

Comment:
Might be better to add a `VCS` prefix so all future params for other providers will be scoped together. A good tip for getting this right is to always try to find existing patterns and extend them, this helps us create both consistency and predictability for both developers and maintainers.